keras/distribution/distribution_lib.py

```diff
@@ -327,10 +327,10 @@ class ModelParallel(Distribution):
     # will be split across 4 devices. Any other variable that doesn't
     # match any key in the layout map will be fully replicated.
     layout_map = LayoutMap(device_mesh)
-    layout_map['.*dense.*kernel'] = TensorLayout([None, 'model'])
-    layout_map['.*dense.*bias'] = TensorLayout(['model'])
-    layout_map['.*conv2d.*kernel'] = TensorLayout([None, None, None, 'model'])
-    layout_map['.*conv2d.*bias'] = TensorLayout(['model'])
+    layout_map['.*dense.*kernel'] = [None, 'model']
```

Comment:
Please use tuples here. In general for static iterables (no appending / popping) tuples should always be prefered.

Reply:
Done.